src/Theme_Command.php

```diff
@@ -436,9 +437,19 @@ protected function get_item_list() {
 				$network_enabled = array();
 		}
 
+		$all_update_info = $this->get_update_info();
+		$checked_themes  = ( isset( $all_update_info->checked ) ) ? $all_update_info->checked : array();
```

Comment:
Superfluous parentheses:
```suggestion
		$checked_themes  = isset( $all_update_info->checked ) ? $all_update_info->checked : array();
```